.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea
 kotlindsl-hazelcast.iml
 target
+config-dsl.iml
```

Comment:
Should replace `kotlindsl-hazelcast.iml` by `*.iml` so that even if an individual developer changes the name, it will catch every IDEA project

Reply:
I understand, I will do that